packaging/os/pacman.py

```diff
@@ -109,11 +109,6 @@
 - pacman: name=baz state=absent force=yes
 '''
 
-import shlex
-import os
-import re
```

Comment:
The import `from ansible.module_utils.basic import *` on line 331 brings in these dependencies, but I guess it's probably better to be explicit. I'll re-add them.